.travis.yml

```diff
@@ -37,7 +37,7 @@ env:
   - TEST_TARGET="ci-fiat-crypto"
   - TEST_TARGET="ci-fiat-parsers"
   - TEST_TARGET="ci-flocq"
-  - TEST_TARGET="ci-hott"
+  - TEST_TARGET="ci-hott" BUILD_TARGET="tools coqbinaries pluginsopt states"
```

Comment:
Build stages are definitely a better solution, they just didn't exist when I made the commit ;)